todo0/prisma/schema.prisma

```diff
@@ -41,10 +41,11 @@ model Organization {
 
 model User {
   id       Int    @id @default(autoincrement())
-  email    String @unique
+  email    String
   name String
   externalId String?
-
+  
+  @@unique([email, orgId, externalId])
```

Comment:
do we really need email in this index?  I would think orgId and externalId are sufficient.  Also the index should probably lead with orgId